Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -541,10 +562,26 @@ extension AVFoundationPlayback {
         return self.options[kMinDvrSize] as? Double ?? 60.0
     }
 
-    open override var usingDVR: Bool {
-        guard playbackType == .live else { return false }
+    open override var isDvrInUse: Bool {
+        if isPaused && isDvrAvailable { return true }
+        guard let end = dvrWindowEnd, playbackType == .live else { return false }
         guard let currentTime = player?.currentTime().seconds else { return false }
-        return seekableTimeRanges.first(where: { $0.timeRangeValue.end.seconds > currentTime}) != nil
+        return end > currentTime
+    }
+
+    open override var isDvrAvailable: Bool {
+        guard playbackType == .live else { return false }
+        var liveDuration: Double = 0
+        
+        for seekableTimeRange in seekableTimeRanges {
+            liveDuration += seekableTimeRange.timeRangeValue.duration.seconds
+        }
+        
+        return liveDuration >= minDvrSize
```

Comment:
Instead of repeating the same calculation done in `duration` we can use the `duration` value here.